sdk/examples/wallet/15_destroy_foundry.rs

```diff
@@ -13,39 +21,43 @@ async fn main() -> Result<()> {
     // This example uses secrets in environment variables for simplicity which should not be done in production.
     dotenvy::dotenv().ok();
 
-    // Create the wallet
-    let wallet = Wallet::builder().finish().await?;
+    // Access the wallet we generated with `--example create_wallet`
+    let wallet = Wallet::builder()
+        .with_storage_path(&var("WALLET_DB_PATH").unwrap())
+        .finish()
+        .await?;
+    let account = wallet.get_account(&var("ACCOUNT_ALIAS_1").unwrap()).await?;
 
-    // Get the account we generated with `01_create_wallet`
-    let account = wallet.get_account("Alice").await?;
     // May want to ensure the account is synced before sending a transaction.
     let balance = account.sync(None).await?;
 
-    // Get the first foundry
+    // We try to destroy the first foundry in the account
     if let Some(foundry_id) = balance.foundries().first() {
-        println!("Balance before destroying:\n{balance:?}",);
+        let foundries_before = balance.foundries();
+        println!("Foundries BEFORE destroying:\n{foundries_before:#?}",);
 
         // Set the stronghold password
         wallet
-            .set_stronghold_password(&std::env::var("STRONGHOLD_PASSWORD").unwrap())
+            .set_stronghold_password(&var("STRONGHOLD_PASSWORD").unwrap())
             .await?;
 
+        println!("Sending the destroy-foundry transaction...");
+
         let transaction = account.destroy_foundry(*foundry_id, None).await?;
         println!("Transaction sent: {}", transaction.transaction_id);
 
         let block_id = account
             .retry_transaction_until_included(&transaction.transaction_id, None, None)
             .await?;
-
         println!(
-            "Block included: {}/block/{}",
-            std::env::var("EXPLORER_URL").unwrap(),
+            "Transaction included: {}/block/{}",
+            var("EXPLORER_URL").unwrap(),
             block_id
         );
 
         let balance = account.sync(None).await?;
-
-        println!("Balance after destroying:\n{balance:?}",);
+        let foundries_after = balance.foundries();
+        println!("Foundries AFTER destroying:\n{foundries_after:#?}",);
     }
```

Comment:
add log when no foundry available like in 14

Reply:
done